README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses])[.withFilter(function)] => transit`
```

Comment:
do you think it would be straightforward to generalise so that these can be used in either order - and shrink down to only one place for creating the public API?